modules/rosa-sts-cluster-terraform-customization.adoc

```diff
@@ -0,0 +1,754 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+//
+ifeval::["{context}" == "rosa-sts-creating-a-cluster-quickly-terraform"]
+:tf-defaults:
+endif::[]
+:_content-type: PROCEDURE
+
+= Creating your ROSA cluster with Terraform
+
+The following Terraform cluster example creates your account-wide IAM roles then creates a ROSA cluster with a managed OIDC configuration.
+
+.Procedure
+. Optional: since the Terraform files that are created during this process are created in your current directory, you can create a new directory to store these files. Run the following command to create and navigate into a new directory:
```

Comment:
```suggestion
. Optional: Because the Terraform files are created in your current directory during this procedure, you can create a new directory to store these files. 
.. To create and navigate into a new directory, run the following command:
```
I would use "because" here. "Since" might translate as a time word ("...since you've been gone...")
There's surely a better link for this, but we cap after the colon of Optional steps (footnote #2) https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks 
I prefer the reverse construction for giving the "run the command" instruction because it is easier scan down the list for the one I want if the thing I am doing is at the beginning of the sentence.